apidocs/src/partials/url_encoding.md

```diff
@@ -0,0 +1,15 @@
+#### Note: entity IDs must be url-encoded
+
+Any identifier included in the URL must be [percent encoded][percent-encoding]
```

Comment:
Can you change the text here to be URL-encoded instead of percent encoded? We should use the same terminology throughout the docs. Everywhere else we say URL-encoded.